contents/handbook/growth/sales/new-hire-onboarding.md

```diff
@@ -0,0 +1,74 @@
+---
+title: New starter onboarding
+sidebar: Handbook
+showTitle: true
+---
+
+## Your first few weeks
+
+Welcome to the PostHog Sales and Customer Success team!  We only hire about 1 in 400 applicants, so you've done well to make it here!  Unlike a lot of companies, we don't have a super-long onboarding process and would prefer you to be up and running with your customer base as quickly as possible.  Here are the things you should focus on in your first few weeks at PostHog to help you achieve that.
+
+### Week 1
+
+#### Day 1
+
+ - Meet with [Simon](/community/profiles/28895) who will run through this plan and answer any questions you may have.  In addition, come equipped to talk about any nuances around how you prefer to work (e.g. schedules, family time etc.).
+ - Meet with [Mine](/community/profiles/29862) who will get you set up with core tooling:
+   - [Salesforce](https://posthog.lightning.force.com/)
+   - [Vitally](https://posthog.vitally-eu.io/)
+   - [Zendesk](https://posthoghelp.zendesk.com/agent/dashboard)
+   - PostHog [US](https://us.posthog.com/) and [EU](https://eu.posthog.com/) instances (and our production project on the US)
+   - [Jimminy](https://app.jiminny.eu/dashboard)
+ - If you start on a Monday, join your first PostHog All Hands (at 4.30pm UK/8.30am PT) and be prepared to have a strong opinion on whether pineapple belongs on pizza.
+ - If you start on a Monday, join your fist Sales and CS standup.
+   - We fill in a GitHub issue every week before this meeting so we are prepared for the discussion topics.  Simon will add your GitHub handle to the template.
+
+#### Rest of week 1
+
+ - Ask team members in your region to be invited to some customer calls so you can gain an understanding of how we work with customers.
+ - Check out some [Jimminy](https://app.jiminny.eu/dashboard) calls.
+ - Start to learn and practise a [demo](https://youtu.be/2jQco8hEvTI) of PostHog.
+ - Read all of the Sales and CS section in the Handbook.
+ - Meet with [Charles](/community/profiles/28625), the exec responsible for Sales and CS.
+
+### Week 2
+
+- During your first week, Simon will figure out your initial book of business (around 10-15 accounts for AEs, 30 for CSMs).  We will review these at the start of your second week.
+- Shadow more live calls and listen to more [Jimminy](https://app.jiminny.eu/dashboard) recordings.
+- Towards the end of the week, schedule a demo and feedback session with Simon.  We might need to do a couple of iterations over the next few weeks as you take onboard feedback, don't worry if that's the case!
+- Review your current book of customers, and prioritize who you want to reach out to first.
+- Get comfortable with the PostHog [Docs](/docs).
+
+#### AE-specific
+
+- We'll start routing new Salesforce Leads to you at the end of week 1.  Start to review these and reach out, using a shared booking link with someone else from your region so they can back you up in the first few weeks.
```

Comment:
Do we have a Cal/Calendly/something else account we should refer to for shared booking links here?